configs/recognition/tsn/tsn_r50_video_320p_1x1x3_100e_kinetics400_rgb.py

```diff
@@ -35,12 +35,7 @@
         num_clips=3,
         start_index=0),
     dict(type='DecordDecode'),
-    dict(
-        type='MultiScaleCrop',
-        input_size=224,
-        scales=(1, 0.875, 0.75, 0.66),
-        random_crop=False,
-        max_wh_scale_gap=1),
+    dict(type='RandomResizedCrop'),
```

Comment:
does it mean that RandomResizedCrop is better than MultiScaleCrop in this case?

Reply:
Sure, we already validated that RandomResizedCrop can outperform MultiScaleCrop. The contribution of that PR is to show that training with videos doesn't lead to any performance drop. 
Before renaming, this config is named `tsn_r50_video_1x1x3_100e_kinetics400_rgb.py`, and there is no performance score for that config in README.